### DISCUSSION TITLE
moving overlay

### DISCUSSION THREAD

**Discussion by bscabl** [YimMenu/YimMenu#1450] [Q&A]
I cant find this on google.. is there a way to move the overlay so it doesnt block the hold menu in gta [move from left top]? i cant find it on google and i like the overlay but want to move it over. thanks in advance!

**Comment by Primexz** [comment]
Open the menu and move it with your mouse. ;)

**Reply by bscabl** [reply]
does this move the overlay?

**Reply by bscabl** [reply]
it doesnt even move the menu .... at least for me

**Reply by Primexz** [reply]
> it doesnt even move the menu .... at least for me

The menu isn‘t movable, the overlay is.
